.pre-commit-config.yaml

```diff
@@ -41,6 +41,7 @@ repos:
             "--docstring-convention=google",
             "--ignore-decorators=overload",
             "--ignore=E203,W503,D105,D107,DAR203",
+            "--extend-ignore=E402,F811 examples",
```

Comment:
Do we need a `/` to point out it is a directory?
`examples/`

Reply:
Seems this command will make flake8 run only on the files in `examples`

Reply:
This command will not run only on the files in examples